lib/libfrr.c

```diff
@@ -1474,3 +1474,11 @@ const char *frr_vers2str(uint32_t version, char *buf, int buflen)
 
 	return buf;
 }
+
+bool frr_is_daemon(void)
+{
+	if (di)
```

Comment:
nit: `return !!di`?

Reply:
I think we are looking at our belly button too much.